scripts/DocParser.js

```diff
@@ -124,6 +125,47 @@ function validate (docs, name, componentDirectory, strict) {
 	}
 }
 
+function parseTableOfContents (frontMatter, body) {
+	let maxdepth;
+	const tocConfig = frontMatter.match(/^toc: ?(false|\d+)$/m);
```

Comment:
I do have some concerns over the overloading of the types.  I'll have to test but I think the graphql schema that is generated doesn't have the appropriate typing so we wouldn't be able to use this data within Gatsby if we needed to in the future.

Reply:
I don't think it'll be available in gatsby as it statnds. We could probably just use 0 or -1 to disable the table of contents.

Reply:
Let's do that.  I think it is better to suppress that warning so we don't get into the habit of ignoring warnings in the build.

Reply:
Changing to `0` does the trick.